app/controllers/tasks_controller.rb

```diff
@@ -1,5 +1,54 @@
 class TasksController < ApplicationController
-  def tasks
-    @tasks = Task.all
+  before_action :set_user
+  before_action :find_task, only: %i[show edit update destroy]
+
+  def index
+    @tasks = @user.tasks
```

Comment:
Let's keep it like this for now. Handle the list of tasks with cancancan afterwards when you implement authorization.

Reply:
Ok thank you then i wont continue with the tasks until authorization is added.